vaadinfrontend/frontend/themes/datamanager/components/card.css

```
@@ -227,3 +226,74 @@
   list-style-type: none;
   display: contents;
 }
+
+/* Support section formatting and components */
+
+.experiment-item-collection {
+  display: grid;
+  row-gap: 1rem;
+  grid-auto-rows: max-content;
+}
+
+/* Experiment item */
+
+.experiment-item {
+  display: block;
+  height: 100px;
+  width: 100%;
+}
+
+.experiment-item:hover {
+  background-color: #f5f5f5;
+  cursor: pointer;
+}
+
+.experiment-item .progress-tag{
+  text-align: center;
+  width: fit-content;
+
+}
+
+.experiment-item .experiment-title {
+  font-weight: bold;
+  padding-left: 1rem;
+  color: #3f3f3f;
+}
+
+.experiment-item .content-section {
+  padding: 1rem 1rem 0 0;
+  display: flex;
+  justify-content: space-between;
+}
+
+.experiment-item .progress-section {
+  background-color: gainsboro;
+  font-size: smaller;
+  padding-left: 1rem;
+}
+
+.experiment-item .progress-section .incomplete{
+  background-color: gainsboro;
+}
+
+.experiment-item vaadin-icon {
+  color: #949494;
+}
+
+.experiment-item .active-section {
+  padding-left: 1rem;
+}
```

I think it would be nice to account for the space the active tag would take already in the card. Otherwise the card size changes dependent on if the active tag is set or not. 
One possibility could be to set a padding or a min-width in the active section 
```suggestion
.experiment-item .active-section {
  padding-left: 1rem;
  padding-bottom: var(--lumo-space-s);
  padding-top: var(--lumo-space-l);
}
```